changelog.md

```diff
@@ -8,6 +8,8 @@
 * **[Squadrons]** Ability to define a livery-set for each squadron from which Retribution will randomly choose during mission generation
 * **[Modding]** Updated support for F/A-18E/F/G mod version 2.2.5
 * **[Modding]** Added VSN F-106 Delta Dart mod support (v2.9.4.101)
+* **[Modding]** Added VSN EA-6B Prowler mod support (v2.9.4.102)
```

Comment:
Missing changelog entry?

Reply:
I don't remember anymore, but it's possible I forgot to add a changelog entry for the EA-6B Prowler mod originally, so I might've included that in this PR.